test/Base.t.sol

```diff
@@ -202,303 +150,88 @@ abstract contract BaseTest is RhinestoneModuleKit, Test {
-        address guardian,
-        address emailRecoveryModule
-    )
-        public
-    {
-        EmailAuthMsg memory emailAuthMsg =
-            getAcceptanceEmailAuthMessage(account, guardian, emailRecoveryModule);
-        IEmailRecoveryModule(emailRecoveryModule).handleAcceptance(emailAuthMsg, templateIdx);
-    }
 
-    // WithAccountSalt variation - used for creating incorrect recovery setups
-    function acceptGuardianWithAccountSalt(
-        address account,
-        address guardian,
-        address emailRecoveryModule,
-        bytes32 optionalAccountSalt
-    )
-        public
-    {
-        EmailAuthMsg memory emailAuthMsg = getAcceptanceEmailAuthMessageWithAccountSalt(
-            account, guardian, emailRecoveryModule, optionalAccountSalt
+        /// Sign 7702 Authorization to delegate to the MSA Account
+        Vm.SignedDelegation memory signedDelegation = vm.signDelegation(
+            erc7579implementation,
+            eoaWallet.privateKey
         );
-        IEmailRecoveryModule(emailRecoveryModule).handleAcceptance(emailAuthMsg, templateIdx);
-    }
 
-    function getAcceptanceEmailAuthMessage(
-        address account,
-        address guardian,
-        address emailRecoveryModule
-    )
-        public
-        returns (EmailAuthMsg memory)
-    {
-        return getAcceptanceEmailAuthMessageWithAccountSalt(
-            account, guardian, emailRecoveryModule, bytes32(0)
-        );
-    }
-
-    // WithAccountSalt variation - used for creating incorrect recovery setups
-    function getAcceptanceEmailAuthMessageWithAccountSalt(
-        address account,
-        address guardian,
-        address emailRecoveryModule,
-        bytes32 optionalAccountSalt
-    )
-        public
-        virtual
-        returns (EmailAuthMsg memory)
-    {
-        string memory command;
-        bytes[] memory commandParamsForAcceptance = new bytes[](1);
-        if (getCommandHandlerType() == CommandHandlerType.AccountHidingRecoveryCommandHandler) {
-            bytes32 accountHash = keccak256(abi.encodePacked(account));
-            string memory accountHashString = uint256(accountHash).toHexString(32);
-            command = string.concat("Accept guardian request for ", accountHashString);
-            commandParamsForAcceptance[0] = abi.encode(accountHashString);
-        } else {
-            string memory accountString = CommandUtils.addressToChecksumHexString(account);
-            command = string.concat("Accept guardian request for ", accountString);
-            commandParamsForAcceptance[0] = abi.encode(account);
-        }
-
-        bytes32 nullifier = generateNewNullifier();
-
-        bytes32 accountSalt;
-        if (optionalAccountSalt == bytes32(0)) {
-            accountSalt = getAccountSaltForGuardian(account, guardian);
-        } else {
-            accountSalt = optionalAccountSalt;
-        }
+        /// Prepare the initData with the bootstrap contract call along with the, can go with the generic one
```

Comment:
Did this sentence get cut short here?

Reply:
Oh yeah, seems like a mistake while comment, will update with any other changes we'll have here